### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by victorokoroji**
_In this milestone, i;_

- Created a Post on behalf of the current_user 
- Created a comment on behalf of the current_user (use the method that you created in your ApplicationController).
- Allowed Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -2,10 +2,35 @@ class PostsController < ApplicationController
...
     @post = Post.includes(:user, comments: [:user]).find(params[:id])
   end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @user = User.find(params[:user_id])
+    @post = Post.new(post_params)
```
Comment (thecodechaser) [inline app/controllers/posts_controller.rb]: - [ ] On this method you are creating the post independently but the post belongs to a user, You should create a post the way it should belong to the current user. Please fix this issue. You can consider using this code `@post = current_user.posts.new(post_params)`.

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -1,2 +1,23 @@
 class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
+    user = User.find(params[:user_id])
+    post = Post.find(params[:post_id])
```
Comment (thecodechaser) [inline app/controllers/comments_controller.rb]: - [ ] On this method you are doing the same as I explained above, Please make sure that you create a comment that belongs to the current_user and the posts you are in.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @vickymarz👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/comments_controller.rb (hunk 2 above)
Comment (victorokoroji) [reply app/controllers/comments_controller.rb]: I have implemented all required changes, thanks

**Review by Mre55 — APPROVED** [review]
**Hi @vickymarz,**

# Status: APPROVED 🏆 🎉 🎊 

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**